src/components/Router/CrossDeviceMobileRouter.tsx

```diff
@@ -234,10 +234,24 @@ export default class CrossDeviceMobileRouter extends Component<
       return this.setError()
     }
 
+    // NOTE, This is should be removed as soon as motion is supported on Mobile
```

Comment:
I added `createCrossDeviceStepsHook` to group crossdevice steps "dynamic" changes.